docs/src/main/java/org/geotools/cql/ECQLExamples.java

```diff
@@ -145,6 +151,25 @@ private static void relatePattern() throws Exception{
 		
 	}
 
+    /**
+     * Example using the "contains" intersection matrix.
+     *
+     * @throws Exception
+     */
+    private static void referencedRelatePattern() throws Exception{
```

Comment:
Should be a space between "Exception" and "{". (Cut and paste from another method in this class, I am sure.)

Reply:
Yep